docs/README.md

```diff
@@ -0,0 +1,80 @@
+# KubeVela Docs
+
+[Kubevela website](http://kubevela.io/)  is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+Whenever you modify the files under docs and submit a PR, the `check-docs` action will verify whether your docs are accurate.
+After PR is merged, files(`docs/en/*`, `resource/*`, `sidebars.js`, `docusaurus.config.js`, `index.js`) will be submitted to the corresponding locations in the repo 
+[kubevela.io](https://github.com/oam-dev/kubevela.io). Finally, github-action will parse the document and publish it to the Kubevela website. 
+
+This document will guide you how to update the KubeVela docs.
+
+## Add or Update Docs
+
+When you add or update docs, you maybe need to fix 3 files: `docs/en/`, `resource/` and `sidebars.js`.
+
+1. `docs/en/`, markdown files are placed in this folder, every markdown file needs to 
+   add title at the beginning in the following format:
+   
+    ```markdown
+    ---
+    title: Title Name
+    ---
+
+    ```
+
+   When you want to add a link refer to `md` file inside `docs/en`, you need to use relative path and remove the `md` suffix.
+   such as `en/helm/component.md` refer to `en/platform-engineers/definition-and-templates.md`. The link should like:
+   
+    ```markdown
+    [the definition and template concepts](../platform-engineers/definition-and-templates)
+    ```
+   
+2. `resource/`, image files are placed in this folder, When you want to use image in md, please also use relative path:
+  
+   ```markdown
+    ![alt](../resources/concepts.png)
+    ```
+
+3. `sidebars.js`, this file contain the navigation information of the KubeVela docs.
+    Please read [doc](https://docusaurus.io/docs/sidebar) to learn how to write sidebar.js.
+    
+    ```js
+       {
+          type: 'category',
+          label: 'Capability References',
+          items: [
+            // This is the path under "docs/end" 
+            'developers/references/README',
+            'developers/references/workload-types/webservice',
+            'developers/references/workload-types/task',
+            ...
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Roadmap',
+          items: [
+            'roadmap',
+          ],
+        },
+        {
+          type: 'doc',
+          id: 'developers/references/devex/faq'
+        }
+    ```
+   
+## Local Development
+
+You can develop website in local, but first install node and yarn.
+Every time you modify the files under docs, you need to re-run the following cmd:
+
+```shell
+make docs-start
+```
+
+## Build in Local
+
+You can build Kubevela website in local to test the correctness of docs, only run the following cmd:
```

Comment:
```suggestion
You can also preview the KubeVela website locally by:
```

Reply:
`make docs-build` can only build to test the correctness of docs
`make docs-start` can preview the KubeVela website locally